packages/prismic/components/PrismicSlice.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div :class="{ [`${className}`]: true, 'slice-container': true }">
+    <div class="slice-element" v-html="renderBlock(slice)" :key="i" v-for="(slice, i) in slices">
+    </div>
+  </div>
+</template>
+
+<script>
+import { usePrismic } from '@vue-storefront/prismic'
+import { ref, reactive } from '@vue/composition-api'
+
+const transformRepeatableBlock = (block) => {
+  return block[Object.keys(block)[0]]
+}
+
+const parseType = (repeatableBlock) => {
+  const block = transformRepeatableBlock(repeatableBlock)
+
+  if (typeof block === 'string' || typeof block === 'number') {
+    return 'text'
+  }
+
+  if (Array.isArray(block)) {
+    return 'html'
+  }
+
+  if (block.dimensions !== undefined) {
+    return 'image'
+  }
+
+  if (block.embed_url !== undefined) {
+    return 'embed'
+  }
+
+  if (block.link_type !== undefined) {
+    return block.kind === 'image' ? 'image' : 'link'
+  }
+
+  return undefined
+}
+
+const transformBlock = (block) => ({
```

Comment:
Looks like `const transformedBlock = transformRepeatableBlock(block)` could save a few calls here

Reply:
Oh, that's an old file and it should be removed at all